src/decode.h

```diff
@@ -8,158 +8,160 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "riscv.h"
+
 /* RISC-V instruction list */
 /* clang-format off */
 #define RISCV_INSN_LIST                    \
```

Comment:
Add comment for the description of the instruction listing. That is, `_(instruction-name, can-branch)`